Formula/goose.rb

```diff
@@ -3,21 +3,30 @@
 class Goose < Formula
   desc "Go Language's command-line interface for database migrations"
   homepage "https://github.com/pressly/goose"
-  url "https://github.com/pressly/goose/archive/v1.0.tar.gz"
-  sha256 "d1b84411dcb5abc211d52de65f1eeaa9ea4a1eeaee0400b3380d9d24e1dc8b1b"
+  url "https://github.com/pressly/goose/archive/v2.0.0.tar.gz"
+  sha256 "ebb5036ce89bfbb8e0594149454293fab6c2639873be824b0746994ab5a8668b"
 
   depends_on "go" => :build
 
+  go_resource "github.com/golang/dep" do
+    url "https://github.com/golang/dep.git",
+        :revision => "c79b048e07eccf76d323d4c2e88d7c6d72ea735f"
+  end
+
   def install
     ENV["GOPATH"] = buildpath
 
+    (buildpath/"src/github.com/pressly/goose").install buildpath.children
     Language::Go.stage_deps resources, buildpath/"src"
-    (buildpath/"src/github.com/pressly").mkpath
-    ln_s buildpath, buildpath/"src/github.com/pressly/goose"
 
-    cd buildpath/"cmd/goose" do
-      system "go", "build", "-o", "./goose", "."
+    cd "src/github.com/golang/dep" do
+      system "go", "install", "github.com/golang/dep/cmd/dep"
+    end
+    ENV.append_path "PATH", buildpath/"bin"
```

Comment:
Instead of changing the path, below just do `system buildpath/"bin/dep", "ensure"`.